CHANGELOG.md

```diff
@@ -14,14 +14,21 @@ CHANGELOG
 - Remove CloudFormation DescribeStacks API call from AWS Batch Docker entrypoint. This removes the possibility of job
   failures due to CloudFormation throttling.
 - Add support for io2 EBS volume type.
+- EFA kernel module now installed also on ARM instances with `alinux2` and `ubuntu1804`
```

Comment:
what about: `Install EFA kernel module on ARM instances when using alinux2 and ubuntu1804`